grails-app/conf/UrlMappings.groovy

```diff
@@ -1,10 +1,9 @@
-import grails.util.Environment
-
 class UrlMappings {
 
     static mappings = {
 
         "/proxy"(controller: "proxy", action: "index")
+        "/sitemap/**"(controller: "sitemap", action: "index")
 
```

Comment:
Sorry @pmbohm  .... I mean is it possible to do the following redirection here instead of SearchController? If yes, then will it make more sense?

search/uuid/something to search?uuid=something&info=true